roles/common/tasks/main.yml

```diff
@@ -1,9 +1,4 @@
 ---
-- name: common | update cache
```

Comment:
What's your thinking here? This isn't an FQCN change.

Reply:
It is redundant to https://github.com/pulibrary/princeton_ansible/blob/i2861_common_fqrn/roles/common/tasks/main.yml#L20 
and otherwise not idempotent.